stdlib/universe/group_except_test.flux

```diff
@@ -21,9 +21,9 @@ inData =
 ,,1,2018-05-22T19:53:26Z,648,io_time,diskio2,host.local,disk2
 ,,1,2018-05-22T19:53:36Z,648,io_time,diskio2,host.local,disk2
 ,,1,2018-05-22T19:53:46Z,648,io_time,diskio3,host.local,disk2
-,,1,2018-05-22T19:53:56Z,648,io_time,diskio3,host.local,disk2
-,,1,2018-05-22T19:54:06Z,648,io_time,diskio3,host.local,disk2
-,,1,2018-05-22T19:54:16Z,648,io_time,diskio3,host.local,disk2
+,,1,2018-05-22T19:53:56Z,1200,io_time,diskio3,host.local,disk2
+,,1,2018-05-22T19:54:06Z,1200,io_time,diskio3,host.local,disk2
+,,1,2018-05-22T19:54:16Z,1200,io_time,diskio3,host.local,disk2
```

Comment:
Formerly, the answer we get in the test could either be a row with either diskio2 or diskio3 when reading from storage. By making diskio3 have a higher value, `max()` should be consistent across envs.